src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/help/Edit-AzureRmRecoveryServicesAsrRecoveryPlan.md

```diff
@@ -1,4 +1,4 @@
-﻿---
+---
```

Comment:
Everything you have updated in comments should be updated in the help files, as this is where it will actually be shown to users.   The easiest way to do that would be to add the comments to the "HelpMessage" parameter attribute, as this will automatically update the help message in the help file when regenerated.  This will help you guys in the future as well.

Reply:
Till now we were maintaining MD files manually. and was not much worried about comment on parameter.
Since the code is public domain we decided to correct the summary.
Going forward we will use help message to define the md files .
If we do it in this review that will drastically increase the volume of Review and we want to take it later.